gradle/libs.versions.toml

```diff
@@ -61,6 +62,8 @@ junit-jupiter-params = { module = "org.junit.jupiter:junit-jupiter-params", vers
 junit-pioneer = { module = "org.junit-pioneer:junit-pioneer", version.ref = "junit-pioneer" }
 opentelemetry-annotations = { module = "io.opentelemetry:opentelemetry-extension-annotations", version.ref = "openTelemetry" }
 restAssured = { module = "io.rest-assured:rest-assured", version.ref = "restAssured" }
+testcontainers = { module = "org.testcontainers:testcontainers", version.ref = "testcontainers" }
```

Comment:
Why do we need TestContainers in the sample?

Reply:
Sample 04 is running in docker compose. We're simulating that in the test case. For that we need testcontainers.